src/SignalR/SignalR/Az.SignalR.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.4.0'
```

Comment:
Please revert the change of the line. We will update the version using automatic tools